build.cake

```diff
@@ -139,6 +139,25 @@ Task("PackageZip")
     Zip(BIN_DIR, File(path), zipFiles);
 });
 
+Task("PackageChocolatey")
+.Does(() =>
+{
+	CreateDirectory("PACKAGE_DIR");
+
+	ChocolateyPack("nunit-project-editor.nuspec", 
+		new ChocolateyPackSettings()
+		{
+			OutputDirectory = PACKAGE_DIR,
+			Files = new []
+			{
+				new ChocolateyNuSpecContent { Source = "LICENSE.txt" },
+				new ChocolateyNuSpecContent { Source = "CHANGES.txt" },
+				new ChocolateyNuSpecContent { Source = BIN_DIR + "nunit-editor.exe", Target = "bin" },
```

Comment:
VERIFICATION isn't necessary when the package is coming from a trusted source. It's helpful when a 3rd party maintainer is embedding software directly into a package as a means of verifying the original. Some software vendors only release to the Chocolatey community package repository and that is fine. I don't know fully what you folks talked about, so I don't know what outcome was decided. Don't let me confuse anything that was decided on though.